### PR TITLE
Some change suggestions

### PR DESCRIPTION
**Pull request by soloman1124**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Nevertheless, the order of the user stories in each group does not reflect the d
 - As a player, I need to be able to see how many points I have earned throughout the game.
 - As a player, I need to be able to replay a game with the same initial hand as a new game if I desire.
 - As a player, I need to be informed when I run out of letters in my hand.
-- As a player, at the end of a game, I need to be able to know what the optimal outcome of the game* is.
+- As a player, at the end of a game, I need to be able to know what the optimal outcome of the game* is. (*BONUS QUESTION* :trophy: )
```
Comment (exchange321) [inline README.md]: I'm not sure what your intention is here

File: README.md [hunk 2]
```diff
@@ -50,4 +50,15 @@ Nevertheless, the order of the user stories in each group does not reflect the d
 - As a system administrator, I would like the players to have an enjoyable experience when interacting with the system.
 
 ## Engineering Requirements
-While it might be slightly advantageous if your solution involves technologies/frameworks/programming languages in our tech stack,you are allowed to use any technology/framework/programming language you prefer or are most familiar with. Regardless of which you eventually use, we expect the code to be in high quality.
+While it might be slightly advantageous if your solution involves technologies/frameworks/programming languages in our tech stack:
+- python
```
Comment (exchange321) [inline README.md]: Breaking the sentence like this makes it hard to read. If you want to add these, you probably need to either restructure the sentence or put them somewhere else.

Comment (exchange321) [reply README.md]: Also, could you capitalize them?

File: README.md [hunk 3]
```diff
@@ -50,4 +50,15 @@ Nevertheless, the order of the user stories in each group does not reflect the d
...
+- javascript (es6)
+- typescript
+- golang
+- java
+- groovy
+- docker
+
+you are welcome to use any technology/framework/programming language you prefer or are most familiar with. Regardless of which you eventually use, we expect the code to be in *high quality*.
```
Comment (exchange321) [inline README.md]: If you want to emphasize "high quality", making it bold is probably better.

File: README.md [hunk 4]
```diff
@@ -50,4 +50,15 @@ Nevertheless, the order of the user stories in each group does not reflect the d
...
+
+We also love marrying up engineering excellence with great UX/UI design, demostrating your tenant in frontend work will definitely be a bounus!
```
Comment (exchange321) [inline README.md]: While it's entirely up to @soloman1124, I personally don't think UI/UX should be here. We have Matthew for UI/UX already, so I think we wouldn't mind if the candidate produces a bad UI. As a result, having this here, we are only stressing people out with things that we don't really care.

Comment (exchange321) [reply README.md]: The frontend work could potentially be made into a user story in Could Have: As a player, I would like to be able to play the game anywhere with any device through an Internet browser.

File: README.md (hunk 2 above)
Comment (soloman1124) [reply README.md]: I personally find list style is easier to read

File: README.md (hunk 4 above)
Comment (soloman1124) [reply README.md]: Well, I think you prefer more formal spec type of writing. But I actually find that long sentence  about the device browser etc is a bit lengthy and not very intuitive to me...

Comment (exchange321) [reply README.md]: Adding on to the UI/UX thing, if @soloman1124 sees it as a secondary skill, I think it usually emerges when a person is creating a UI, like Andrew Hill and Vivian. So it isn't necessary unless we want the candidate to show off their UI/UX skills and expect them to do some design work when they get hired.

File: README.md (hunk 2 above)
Comment (exchange321) [reply README.md]: Yes, I agree that a list is easier to read, but not in the middle of a sentence though.

File: README.md (hunk 4 above)
Comment (soloman1124) [reply README.md]: I also don’t feel it is an requirement, it’s just a hint if they want to do more work to impress us

File: README.md (hunk 1 above)
Comment (soloman1124) [reply README.md]: to emphasise if they do spend time on it, I am happy

File: README.md (hunk 4 above)
Comment (exchange321) [reply README.md]: For the browser thingy, I think it's more like the curse of knowledge... Terms like "tenant" and "front-end work" may not necessarily make sense to every graduate developers. If we expect the candidate to have web development experience, perhaps we can paraphrase it to "As a player, I would like to be able to play the game on a website."

**Review by exchange321 — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -48,6 +47,14 @@ Nevertheless, the order of the user stories in each group does not reflect the d
 - As a system administrator, I would like to be able to see a history of all played games.
 - As a system administrator, I would like to be able to see a particular player's history of all played games.
 - As a system administrator, I would like the players to have an enjoyable experience when interacting with the system.
+- As a player, I would like to be able to play the game on a website.
```
Comment (exchange321) [inline README.md:49]: Can you move this to just below the first one? After "As a player, I would like to be able to see my history of all played games."

**Review by exchange321 — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -37,7 +36,7 @@ Nevertheless, the order of the user stories in each group does not reflect the d
 - As a player, I need to be able to see how many points I have earned throughout the game.
 - As a player, I need to be able to replay a game with the same initial hand as a new game if I desire.
 - As a player, I need to be informed when I run out of letters in my hand.
-- As a player, at the end of a game, I need to be able to know what the optimal outcome of the game* is.
+- As a player, at the end of a game, I need to be able to know what the optimal outcome of the game* is. (**BONUS QUESTION** :trophy: )
```
Comment (exchange321) [inline README.md]: There is an extra space at the end

Comment (exchange321) [reply README.md]: Or maybe just add one more space before "BONUS". It looks a bit cramped.

**Review by exchange321 — APPROVED** [review]

**Pull request merged**